frontend/package.json

```diff
@@ -12,12 +12,14 @@
   },
   "dependencies": {
     "@ionic/react": "^7.6.2",
+    "@reduxjs/toolkit": "^2.2.1",
     "bootstrap": "^5.3.2",
     "ionicons": "^7.2.2",
     "react": "^18.2.0",
     "react-bootstrap": "^2.9.1",
     "react-confirm": "^0.3.0-7",
```

Comment:
I'm always reluctant to add new libraries, we really tried to keep the amount of libraries used as low as possible.
I understand that you're used to Ionic and it will help you work better, and having a good library for common components is always useful, so be it. Though, keep in mind that it's a strong choice, as not everyone is used to it.
As soon as possible, it will be better to get rid of `bootstrap`, `react-bootstrap` and `react-confirm` and replace is with either pure CSS or Ionic components.

Reply:
Thanks for the reminder around adding new libraries.
It is my purpose to get rib of these libraries overtime. I'm just being careful by not doing it all right now